### ISSUE TITLE
About the learning rate schedule for finetuning stage

### ISSUE THREAD

**Issue by JoyHuYY1412** [arthurdouillard/incremental_learning.pytorch#21]
I notice that the code you released used an additional finetuning stage, which is not mentioned in your paper. Can I simply remove this part and get the result as u post on paper?

**Comment by JoyHuYY1412** [comment]
What's more, it seems you used rebuffi as your backbone, is it matters if I use resnet32?

**Comment by arthurdouillard** [comment]
1. The finetuning learning rate (0.05) can be found in the options files. For example https://github.com/arthurdouillard/incremental_learning.pytorch/blob/master/options/podnet/podnet_cnn_cifar100.yaml#L35

2. We do use a finetuning. You're right we could have been clearer in the paper, sorry about that. Since PODNet based on UCIR we re-use their finetuning method (we mention that in the related work).

3. Not using finetuning shouldn't change much the performance of NME mode, but it will degrade the performances of CNN mode. Note that in the main training part (before the finetuning), the old weights are kept frozen (https://github.com/arthurdouillard/incremental_learning.pytorch/blob/master/options/podnet/podnet_cnn_cifar100.yaml#L30).

4. My "rebuffi" backbone is a translation from Rebuffi's iCaRL backbone in Theano to PyTorch. It shouldn't matter much if you use the default "resnet32".

**Comment by JoyHuYY1412** [comment]
> 1. The finetuning learning rate (0.05) can be found in the options files. For example https://github.com/arthurdouillard/incremental_learning.pytorch/blob/master/options/podnet/podnet_cnn_cifar100.yaml#L35
> 2. We do use a finetuning. You're right we could have been clearer in the paper, sorry about that. Since PODNet based on UCIR we re-use their finetuning method (we mention that in the related work).
> 3. Not using finetuning shouldn't change much the performance of NME mode, but it will degrade the performances of CNN mode. Note that in the main training part (before the finetuning), the old weights are kept frozen (https://github.com/arthurdouillard/incremental_learning.pytorch/blob/master/options/podnet/podnet_cnn_cifar100.yaml#L30).
> 4. My "rebuffi" backbone is a translation from Rebuffi's iCaRL backbone in Theano to PyTorch. It shouldn't matter much if you use the default "resnet32".
> 
> (btw on an unrelated note, I'm a big fan of your LST paper!)

Wow, thank you for your detailed explanation. I do admire your work and the code!
